catalog/app/components/Preview/loaders/Audio.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+
+import * as AWS from 'utils/AWS'
+import AsyncResult from 'utils/AsyncResult'
+import * as Config from 'utils/Config'
+import type { S3HandleBase } from 'utils/s3paths'
+
+import { PreviewData } from '../types'
+
+import * as utils from './utils'
+
+export const detect = utils.extIn([
+  '.flac',
+  '.m2t',
+  '.mp3',
+  '.mp4',
```

Comment:
also, wont this conflict with video (since it's a multipurpose container)?

Reply:
Yes, thanks
`.m2t` too